its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -47,7 +47,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class TestUtils {
-  public static final String MSBUILD_PATH = "msbuild.path";
+  public static final String MSBUILD_PATH = "MSBUILD_PATH";
```

Comment:
If you change that, you should also update cix.sh

Reply:
Ah good point! I made this change because for SonarC# and SonarVB the env variable was not set but we already had MSBUILD_PATH one set. I assumed it was the same here. Actually do we really need the `-Dmsbuild.path` property? It seems like `orchestrator.getConfiguration().getString()` will pick up Jenkins/Machine (not sure how it is set up by builders) environment variable (this is what happens for SonarC# and SonarVB).

Reply:
Yep, I think Configuration contains the env. The point of having a property is to support overriding the env. Maybe you could use 2 fallbacks:
`orch.getConfiguration().getString("msbuild.path", orch.getConfiguration().getString("MSBUILD_PATH", "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe"));`